dev/reference/performance/statement-summary.md

```diff
@@ -23,48 +22,145 @@ SELECT * FROM employee WHERE id IN (1, 2, 3) AND salary BETWEEN 1000 AND 2000;
-      SUM_LATENCY: 1035161
-      MAX_LATENCY: 399594
-      MIN_LATENCY: 301353
-      AVG_LATENCY: 345053
-SUM_ROWS_AFFECTED: 0
-       FIRST_SEEN: 2019-09-12 18:47:14
-        LAST_SEEN: 2019-09-12 18:47:16
-QUERY_SAMPLE_TEXT: select * from employee where id=3100
+   SUMMARY_BEGIN_TIME: 2020-01-02 11:00:00
+     SUMMARY_END_TIME: 2020-01-02 11:30:00
+            STMT_TYPE: select
+          SCHEMA_NAME: test
+               DIGEST: 0611cc2fe792f8c146cc97d39b31d9562014cf15f8d41f23a4938ca341f54182
+          DIGEST_TEXT: select * from employee where id = ?
+          TABLE_NAMES: test.employee
+          INDEX_NAMES: NULL
+          SAMPLE_USER: root
+           EXEC_COUNT: 3
+          SUM_LATENCY: 1035161
+          MAX_LATENCY: 399594
+          MIN_LATENCY: 301353
+          AVG_LATENCY: 345053
+    AVG_PARSE_LATENCY: 57000
+    MAX_PARSE_LATENCY: 57000
+  AVG_COMPILE_LATENCY: 175458
+  MAX_COMPILE_LATENCY: 175458
+  ...........
+              AVG_MEM: 103
+              MAX_MEM: 103
+    AVG_AFFECTED_ROWS: 0
+           FIRST_SEEN: 2020-01-02 11:12:54
+            LAST_SEEN: 2020-01-02 11:25:24
+    QUERY_SAMPLE_TEXT: select * from employee where id=3100
+     PREV_SAMPLE_TEXT:
+          PLAN_DIGEST: f415b8d52640b535b9b12a9c148a8630d2c6d59e419aad29397842e32e8e5de3
+                 PLAN:  Point_Get_1     root    1       table:employee, handle:3100
 ```
 
-以下是各个字段的含义：
-
-| 列名           | 含义                      |
-|:----------------- |:-------------------------------- |
-| SCHEMA_NAME       | 执行这类 SQL 的当前 schema |
-| DIGEST            | SQL 的 digest                   |
-| DIGEST_TEXT       | 规范化后的 SQL              |
-| EXEC_COUNT        | 这类 SQL 执行的总次数    |
-| SUM_LATENCY       | 这类 SQL 执行的总延迟，单位 ns |
-| MAX_LATENCY       | 这类 SQL 执行的最大延迟，单位 ns |
-| MIN_LATENCY       | 这类 SQL 执行的最小延迟，单位 ns |
-| AVG_LATENCY       | 这类 SQL 执行的平均延迟，单位 ns |
-| SUM_ROWS_AFFECTED | 这类 SQL 的总影响行数    |
-| FIRST_SEEN        | 这类 SQL 第一次执行的时间 |
-| LAST_SEEN         | 这类 SQL 最后一次执行的时间 |
-| QUERY_SAMPLE_TEXT | 这类 SQL 首次出现的原 SQL 语句 |
+> **注意：**
+>
+> 在 TiDB 中，statement summary tables 中字段的时间单位是纳秒 (ns)，而 MySQL 中的时间单位是皮秒 (ps)。
+
+### 表的字段介绍
+
+SQL 的基础信息：
+
+- `STMT_TYPE`：SQL 语句的类型
+- `SCHEMA_NAME`：执行这类 SQL 的当前 schema
+- `DIGEST`：这类 SQL 的 digest
+- `DIGEST_TEXT`：规一化后的 SQL
+- `QUERY_SAMPLE_TEXT`：这类 SQL 的原 SQL 语句，多条语句只取其中一条
+- `TABLE_NAMES`：SQL 中涉及的所有表，多张表用 `,` 分隔
+- `INDEX_NAMES`：SQL 中使用的索引名，多个索引用 `,` 分隔
+- `SAMPLE_USER`：执行这类 SQL 的用户名，多个用户名只取其中一个
+- `PLAN_DIGEST`：执行计划的 digest
+- `PLAN`：原执行计划，多条语句只取其中一条的执行计划
+
+执行时间相关的信息：
+
+- `SUMMARY_BEGIN_TIME`：当前统计的时间段的开始时间
+- `SUMMARY_END_TIME`：当前统计的时间段的结束时间
+- `FIRST_SEEN`：这类 SQL 的首次出现时间
+- `LAST_SEEN`：这类 SQL 的最后一次出现时间
+
+在 TiDB server 上的执行数据：
+
+- `EXEC_COUNT`：这类 SQL 的总执行次数
+- `SUM_LATENCY`：这类 SQL 的总延时
+- `MAX_LATENCY`：这类 SQL 的最大延时
+- `MIN_LATENCY`：这类 SQL 的最小延时
+- `AVG_LATENCY`：这类 SQL 的平均延时
+- `AVG_PARSE_LATENCY`：解析器的平均延时
+- `MAX_PARSE_LATENCY`：解析器的最大延时
+- `AVG_COMPILE_LATENCY`：优化器的平均延时
+- `MAX_COMPILE_LATENCY`：优化器的最大延时
```

Comment:
sorry just remember to ask, what's the unit for these time metrics, I guess it should be "ms" or "us", right? can we put the units for all metrics?